azure-pipelines.yml

```diff
@@ -146,6 +146,7 @@ jobs:
         runCategories: 'runtime libraries' 
         channels:
           - main
+          - nativeaot10.0
```

Comment:
Also if we can remove this line, we will add it once we are ready to support it

Reply:
```suggestion
```